src/blockstack_cli.rs

```diff
@@ -286,6 +293,36 @@ fn handle_contract_call(args: &[String], version: TransactionVersion) -> Result<
     Ok(to_hex(&signed_tx_bytes))
 }
 
+// const TRANSFER_USAGE: &str = "blockstack-cli (options) transfer [origin-secret-key-hex] [fee-rate] [nonce] [recipient-address] [amount] [args...]
+
+fn handle_token_transfer(args: &[String], version: TransactionVersion) -> Result<String, CliError> {
+    if args.len() >= 1 && args[0] == "-h" {
+        return Err(CliError::Message(format!("USAGE:\n {}", TOKEN_TRANSFER_USAGE)))
+    }
+    if args.len() < 5 {
+        return Err(CliError::Message(format!("Incorrect argument count supplied \n\nUSAGE:\n {}", CALL_USAGE)))
+    }
+    let sk_origin = StacksPrivateKey::from_hex(&args[0])?;
+    let fee_rate = args[1].parse()?;
+    let nonce = args[2].parse()?;
+    let recipient_address = StacksAddress::from_string(&args[3]).ok_or("Failed to parse contract address")?;
+    let amount = &args[4].parse()?;
+    let memo = TokenTransferMemo([0u8; 34]);
```

Comment:
This should probably be a CLI argument.  Lots of token-transfers in Stacks v1 use this field.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1259/commits/66cdef1087b0db9208dd01fa7ff47bca2ad67903